lua/entities/gmod_wire_expression2/core/custom/sv_coroutines.lua

```diff
@@ -24,53 +24,56 @@ vex.registerExtension("coroutines", false, "Allows E2s to use coroutines.")
 -- Coroutine Object handling
 
 registerType("coroutine", "xco", nil,
-	nil,
```

Comment:
No code is changed.
I think we should be using spaces for indentation.
These are tabs, so I've reformatted all the touched files to use spaces.
Be sure to configure your editor properly, so this mix doesn't happen again. (I use Visual Studio Code and Sublime Text 4.)

Reply:
I was actually using spaces before seeing this, do you mean using tabs instead? I had no idea vscode auto-converted tabs to spaces, shows how much of a noob I am. Switched to tabs for now

Reply:
No. Use spaces.